extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -141,6 +141,20 @@ public void pause() {
         }
     }
 
+    @Override
+    public void pause(String identity) {
+        if (identity == null) {
+            LOGGER.warn("Cannot pause - identity is null");
+            return;
+        }
```

Comment:
We should also check for `empty string` and ignore them. 

I also think this should use the https://github.com/quarkusio/quarkus/blob/c573fbee41915278e71475f5bf9ed276aaf26b42/extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/util/SchedulerUtils.java#L69 as the identity can also be an expression.

Reply:
Also checking for empty string now